keps/sig-node/1898-hardened-exec/README.md

```diff
@@ -307,64 +323,31 @@ send both query & body parameters, both old and new Kubelets will accept the req
 
 ## Production Readiness Review Questionnaire
 
-<!--
-
-Production readiness reviews are intended to ensure that features merging into
-Kubernetes are observable, scalable and supportable; can be safely operated in
-production environments, and can be disabled or rolled back in the event they
-cause increased failures in production. See more in the PRR KEP at
-https://git.k8s.io/enhancements/keps/sig-architecture/20190731-production-readiness-review-process.md.
-
-The production readiness review questionnaire must be completed for features in
-v1.19 or later, but is non-blocking at this time. That is, approval is not
-required in order to be in the release.
-
-In some cases, the questions below should also have answers in `kep.yaml`. This
-is to enable automation to verify the presence of the review, and to reduce review
-burden and latency.
-
-The KEP must have a approver from the
-[`prod-readiness-approvers`](http://git.k8s.io/enhancements/OWNERS_ALIASES)
-team. Please reach out on the
-[#prod-readiness](https://kubernetes.slack.com/archives/CPNHUMN74) channel if
-you need any help or guidance.
-
--->
-
-TODO
-
 ### Feature Enablement and Rollback
 
 _This section must be completed when targeting alpha to a release._
 
 * **How can this feature be enabled / disabled in a live cluster?**
-  - [ ] Feature gate (also fill in values in `kep.yaml`)
-    - Feature gate name:
-    - Components depending on the feature gate:
-  - [ ] Other
-    - Describe the mechanism:
-    - Will enabling / disabling the feature require downtime of the control
-      plane?
-    - Will enabling / disabling the feature require downtime or reprovisioning
-      of a node? (Do not assume `Dynamic Kubelet Config` feature is enabled).
+  - Feature gate `HardenedExecRequests`
+    - Components depending on the feature gate: kube-apiserver
+    - Description: Guards new backwards-incompatible requirements on pod exec requests to the
+      kube-apiserver
+  - Feature gate: `DeprecatedKubeletStreamingAPI`
+    - Components depending on the feature gate: kubelet
+    - Description: Enables the unused (by kube-apiserver) kubelet streaming APIs. Default disabled.
 
 * **Does enabling the feature change any default behavior?**
-  Any change of default behavior may be surprising to users or break existing
-  automations, so be extremely careful here.
+  Yes, enabling `HardenedExecRequests` alters the pod exec API by adding additional constraints.
```

Comment:
I would add a line indicating that the only anticipated client of the kubelet API is the kube-apiserver. Changing an API is less scary when there is only one valid consumer.